shuriken/mjolnir/include/mjolnir/MjolnIROps.td

```diff
@@ -306,6 +308,7 @@ def InvokeOp : MjolnIR_Op<"invoke", [
     }];
 
     let arguments = (ins
+        FlatSymbolRefAttr:$calleeClass, // called method's class
```

Comment:
I will make a change on this instruction on its own branch and PR for this. Let's try to work in fixes for each instruction in separated issues, and having an explanation so we can have proper discussions about why we apply some changes.